src/shared/bg_public.h

```diff
@@ -818,7 +821,8 @@ enum dynMenu_t
   MN_H_NOREACTOR,
   MN_H_NOBP,
   MN_H_NOTPOWERED,
-  MN_H_ONEREACTOR
+  MN_H_ONEREACTOR,
+  MN_H_NOMOREDRILLS
 };
```

Comment:
for the record, it's not a bad idea to add a `,` at the end of the line to avoid diff noise. Or to keep the list dsorted. (nothing blocking)

Reply:
ack.

Reply:
+1 to the comma.